walkeeper/src/replication.rs

```diff
@@ -244,7 +245,7 @@ impl ReplicationConn {
                 None => {
                     // Open a new file.
                     let segno = start_pos.segment_number(wal_seg_size);
-                    let wal_file_name = XLogFileName(timeline, segno, wal_seg_size);
+                    let wal_file_name = XLogFileName(PG_TLI, segno, wal_seg_size);
```

Comment:
Are there any reasons left why we would want to retain a timeline ID in the file name?

Reply:
Yes, at least for consistency to have 1 everywhere. I'd expect 0 break various tooling, for example pg_waldump assumes timeline 1 by default and will search for such files.

Reply:
Consistency with PG's naming in that case? Because we could reasonably use our own naming scheme for XLOG segment files, one without TLI.

But, compatibility sounds like a good enough reason.